app/server/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "lint": "eslint . --cache",
     "prebuild": "yarn && rm -rf build",
-    "build": "babel-node ../node_modules/.bin/webpack",
+    "build": "babel-node --presets es2015 $(npm bin)/webpack",
```

Comment:
> if I remember correctly, you can't opt out of using it, so having two different configs for babel and webpack babel for instance is impossible

You can in the CLI using ` --no-babelrc`, but I see your point. Lets leave as is.